docker/toolbox/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:14.15.4
+ENV PATH="${PATH}:/usr/local/go/bin:/home/node/go/bin"
+ENV CGO_ENABLED=0
+RUN apt-get install git curl && curl -O https://dl.google.com/go/go1.14.4.linux-amd64.tar.gz && tar -xvf go1.14.4.linux-amd64.tar.gz && chown -R root:root ./go && mv go /usr/local
+
+# This section help ingest example data from Druid website to local instance
```

Comment:
why not running those commands from the magefile target ?
mage env:provision or so ?